server/src/main/java/org/opentosca/toscana/api/model/PropertyWrap.java

```diff
@@ -42,7 +42,8 @@ public String getKey() {
     @ApiModelProperty(
         required = true,
         notes = "The \"Datatype\" of a property. Supported Types are \"name\", \"text\", \"boolean\", " +
-            "\"integer\", \"unsigned_integer\", \"float\" and \"secret\"",
+            "\"integer\", \"unsigned_integer\", \"float\" and \"secret\". If you try to set a value of a non existant key." +
```

Comment:
You might use single quote characters for quoting inside java string, makes these strings much more readable.